.github/workflows/ruby_test.yml

```diff
@@ -86,21 +86,21 @@ jobs:
       - name: Setup node
         uses: actions/setup-node@v2
         with:
-          node-version: "16"
-      - name: Cache js
-        uses: actions/cache@v2
-        with:
-          path: |
-            public/assets
-            node_modules
-          key: ${{ runner.os }}-tests-${{ hashFiles('yarn.lock') }}
-          # If we don't find the specific cache we want, fallback to the last raketests
-          # cache, then finally any cache for this repo.
-          # Github looks for the newest cache beginning with the first entry, before
-          # falling back the the second if none is present.
-          restore-keys: |
-            ${{ runner.os }}-tests-
-            ${{ runner.os }}-
+          node-version: "18"
```

Comment:
You can refer to the .nvmrc file rather than setting the node version in here which saves changing it every time.